src/pages/CreateEventPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CreateEvent from '../components/CreateEvent'
+
+function CreateEventPage() {
+  return (
+    <>
+      <CreateEvent />
+    </>
+  );
+}
```

Comment:
This page component is weird, you usually wouldn't want to delegate everything onto a "general" component. See my other comment in the `CreateEvent` file.